sympy/concrete/tests/test_guess.py

```diff
@@ -0,0 +1,51 @@
+from sympy.concrete.guess import (
+            find_simple_recurrence_vector,
+            find_simple_recurrence,
+            rationalize,
+            guess_generating_function_rational,
+            guess_generating_function
+        )
+from sympy import Function, Symbol, sympify
+from mpmath import fib
```

Comment:
You should use sympy's `fibonacci()` function. This way you won't need to `int(fib(n)`, because `fibonacci()` returns an `Integer`.